charts/tidb-operator/templates/scheduler-deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       serviceAccount: {{ .Values.scheduler.serviceAccount }}
     {{- end }}
       containers:
-      - name: tidb-scheduler
+      - name: {{ .Values.scheduler.schedulerName }}
```

Comment:
Container name is used internally, using `tidb-scheduler` is handy for `kubectl exec` and `kubectl logs`. So it's unnecessary to change this.

Reply:
I think it's better to keep the name consistent, and it's not necessary to fix the container name to executing `kubectl exec` and `kubectl logs`

Reply:
If using `tidb-scheduler` as container name, the two container names are always `tidb-scheduler` and `kube-scheduler`. When debugging we can always use the same name without thinking what it is. Besides container name should reflect the actual image it uses other than using the installer specified name.

Reply:
If you want to use `tidb-scheduler` as container name, you can keep the schedulerName as default. But if you want to change the default value then keeping the container name consistent with it makes sense.